### ISSUE TITLE
Add Notice to README and Description

### ISSUE THREAD

**Issue by mkormendy** [jghaanstra/com.xiaomi-miio#189]
I think one of the key important notes about this app is that it requires the Mi Home Gateway to operate. It should say that in the first line of the description. Currently, it's not immediately clear, and it would save someone some time if it was stated in the beginning in the first line of the description.

**Comment by jghaanstra** [comment]
The app supports WiFI devices which do not require a gateway and Zigbee devices that do require a gateway. All of this is state in the first two lines of the description and read me

> Use [Homey](https://www.athom.com/) to control WiFi devices which connect trough the Xiaomi Mi Home app. You can also use this app to control Xiaomi / Aqara Zigbee devices which are connected to a Xiaomi gateway.

What more do you want?

**Comment by mkormendy** [comment]
My apologies. I guess it's confusing for me with the instructions on the Homey app itself.

**Comment by mkormendy** [comment]
So, I have attempted to add my WIFI Roborock S5 (reported as working in the supported devices) to Homey and have had no luck, which is why I figured this needed the gateway or something.
![Screenshot 2023-04-03 at 4 40 04 PM](https://user-images.githubusercontent.com/1437667/229634162-df07800b-8457-4f39-80ea-d03ff63f02c9.png)
![IMG_9748](https://user-images.githubusercontent.com/1437667/229634566-297a3ef3-d2fe-4a0b-80d9-626e9832f0dd.PNG)
What could be the problem?

**Comment by jghaanstra** [comment]
Handshake error means you are not using the correct token. Try and retrieve it again. I'm closing this. If you need further support, please use the support forum.

**Comment by mkormendy** [comment]
I managed to get this to work.

1. First I locked my Roborock S5 to an IP address on my network with my router.
2. Then I removed the Roborock S5 from Mi Home and then re-added it. It maintained the same IP address.
4. Then I used the Get Mi Home Devices Token app to get the latest Token. At first it timed out on that app, but I tried it again, and it worked and produced a new Token.
5. I then entered that new token into my Homey device details for a Roborock Advanced Vacuum Cleaner device under Xiaomi Mi Home selection.
6. It finally succeeded.
